templates/docs/patterns/notification.md

```diff
@@ -90,7 +90,40 @@ View example of the time notification pattern
 
 ## Accessibility
 
-When adding notifications dynamically, it's important that the content of the notification is announced to users using assistive technology. If the notification is urgent, add `aria-live="assertive"` to the element, which will prompt assistive technology to announce it immediately, or use `aria-live="polite"`, which will cause assistive technology to wait for a pause before announcing the information.
+### How it works
+
+Notifications are messages used to communicate information and feedback to the user. Changes in content which are not interactive should be marked as live regions which are denoted by the `aria-live` attribute.
+
+When the notification is time sensitive or critical `aria-live= "assertive"`, or its equivalent `role="alert"`, should be used. Regions specified as assertive will be prioritized by assistive technologies and could potentially clear the speech queue of previous updates. For this reason, it should be used sparingly. Alternatively, when the notification is important, but not urgent, `aria-live="polite"` should be used. This will notify the user of updates at the next graceful opportunity, without interrupting the current task.
+
+In dismissible notifications, an `aria-controls` attribute on the close button references the id on the parent div containing the notification. This makes it clear to screen reader users what the close button is targeting.
+
+Unless the alert from the screen reader is meant to break the user’s workflow, it is important that the notification does not affect keyboard focus.
+
+Notifications should not disappear automatically.
```

Comment:
Could you add these final two points to the considerations section please?

Reply:
Done!